app/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -338,6 +340,10 @@ private IdentifierHolder saveParticipant(Study study, Set<Roles> callerRoles, St
         if (study.isExternalIdValidationEnabled()) {
             options.remove(EXTERNAL_IDENTIFIER);
         }
+        // You can't update time zone after it is set.
```

Comment:
All repeating schedules require a time portion and that schedule will be given in the time zone passed at the time activities are requested. So it will change. But the calculation will always proceed from the event, to the current day, using the first time zone, so the first day is correct (and not delayed to the next day, as we see in the evening here in PST).